build.gradle

```diff
@@ -30,7 +30,7 @@ dependencies {
 	compileOnly "com.google.code.findbugs:jsr305:3.0.2"
 
 
-	modRuntime ("supercoder79:databreaker:0.2.8") {
+	modRuntimeOnly ("supercoder79:databreaker:0.2.8") {
```

Comment:
This should be `modLocalRuntime` imo

If for any reason this mod gets published to a maven, databreaker should not be transitive, making it local would make sense, *to me* at least.

Reply:
It said in the console when I 'synced' the project to use modRuntimeOnly instead of modRuntime.

Reply:
yeah, but I don't think it's a good behavior imo.

With `modRuntimeOnly` the dependency will be marked as transitive in generated maven POMs (aka, should be downloaded too by the one who depends on ecotones)
Databreaker is quite a dangerous mod due to its nature.
`modLocalRuntime` is a valid dependency annotation too, which is not transitive, meaning it won't appear in the generate maven POMs and be truly local, and could avoid some bad surprises in some dev environments.